### ISSUE TITLE
Universal PBR Baking

### ISSUE THREAD

**Issue by Hopefullyidontgetbanned** [animate1978/MB-Lab#193]
As of Current MB-Lab does not Properly support standard blender baking features, a baking option should be  implemented  that will export all inputs as PBR maps that Game Engines can use or what Blender's Principled Shader can use. This would fix #55 and make graphical quality in game engines much better. As well as make MB-Lab easier for users.

**Comment by Hopefullyidontgetbanned** [comment]
I made this a simple flow chart to explain it more simply.
![MB_LAB_PBR_BAKING](https://user-images.githubusercontent.com/45346421/68733185-b6e04c00-058a-11ea-95ec-4ec140526089.png)

**Comment by animate1978** [comment]
Been checking out the code from this repository

https://github.com/danielenger/Principled-Baker

Trying to see if we could adopt some of the code itself.

**Comment by Hopefullyidontgetbanned** [comment]
Likely doesn't need much changing to be honest, it just so happens that blender's native baking feature are a bit limited when it comes to custom shaders.

**Comment by Hopefullyidontgetbanned** [comment]
bump maps might also be neccesary

**Comment by Hopefullyidontgetbanned** [comment]
yesterday while using an experimental principled bdsf shader, i checked out the principled baker plugin and it properly bakes the skin shader

**Comment by lebrewer** [comment]
This would be an awesome feature! Specially because it would allow us to paint skin details, such as hair and veins and scars, directly in Blender and then just bake out to game engines.

**Comment by Hopefullyidontgetbanned** [comment]
PBR Baking is not necessary anymore since MB-Lab's codebase has no more growth left. Also Blender's native baking has had some small improvements in the last 5 years.
